README.md

```diff
@@ -32,7 +38,7 @@
 1. Download our private keys from the `subscriptions-private` S3 bucket. If you have the AWS CLI set up you can run:
 
     ```
-    aws s3 cp s3://subscriptions-private/DEV/subscriptions-frontend.conf /etc/gu
+    aws s3 cp s3://subscriptions-private/DEV/subscriptions-frontend.conf /etc/gu  --profile membership
```

Comment:
Valid comment. If we had a default profile then we don't need to supply the profile explicitly as 'membership'. Currently the credentials file has [membership] so using that